types/zpl-image/index.d.ts

```diff
@@ -0,0 +1,35 @@
+// Type definitions for zpl-image 0.2
```

Comment:
This package must have been generated via an old `dts-gen`. You'll have to match the current package format (I would recommend reading the README and checking out other packages.)